Sources/AppKitBackend/AppKitBackend.swift

```diff
@@ -1724,6 +1758,60 @@ final class NSCustomTapGestureTarget: NSView {
     }
 }
 
+final class NSCustomHoverTarget: NSView {
+    var hoverChangesHandler: ((Bool) -> Void)? {
+        didSet {
+            if hoverChangesHandler != nil && trackingArea == nil {
+                let options: NSTrackingArea.Options = [
+                    .mouseEnteredAndExited,
+                    .activeInKeyWindow,
+                ]
+                let area = NSTrackingArea(
+                    rect: self.bounds,
+                    options: options,
+                    owner: self,
+                    userInfo: nil)
+                addTrackingArea(area)
+                trackingArea = area
+            } else if hoverChangesHandler == nil, let trackingArea {
+                // should be impossible at the moment of implementation
+                // keeping it to be save in case of later changes
+                removeTrackingArea(trackingArea)
+                self.trackingArea = nil
+            }
+        }
+    }
+
+    private var trackingArea: NSTrackingArea?
+
+    override func updateTrackingAreas() {
```

Comment:
Is it possible to modify the existing tracking area instead of removing it and creating a new one? If not, move this tracking area creation into a separate function so that it's not duplicated between updateTrackingAreas and the didSet above